### PR TITLE
Added instructions for Windows 8 as per issue #232

### PR DESCRIPTION
**Pull request by demar42**
I haven't added screenshots, if dlech's could be used from issue ev3dev/ev3dev#232, that would be great. Also, I'm not very good at formatting using this .md file so it may need some cleanup.

### CODE REVIEW

**Comment by mob41** [comment]
Your solution is not usable. Although we can access ssh from the computer. We couldn't ping to Google DNS (8.8.8.8).

If this only happens on me, I'm sorry
Image

**Comment by dlech** [comment]
@mob41, this tutorial is for "tethering" rather that "connecting to the Internet" it is expected that you will not be able to connect to the Internet by following these steps.

File: docs/tutorials/using-bluetooth-tethering.md [hunk 1]
```diff
@@ -135,11 +135,26 @@ If you are using an older version, please upgrade.
...
+            status bar also indicates that Bluetooth is powered on.
+    
+            {% include screenshot.html source="/images/brickman/bluetooth-powered-visible-no-devices.png" %}
+    
+        2.  Go back to *Wireless and Networks* and select *Tethering*. Enable tethering
+            via Bluetooth by checking the box next to *Bluetooth*.
+    
+            {% include screenshot.html source="/images/brickman/tethering-bluetooth-enabled.png" %}
+        
+        3.  Go to the charms menu and search for *Bluetooth Settings*. Click on the one that will take you to PC Settings.
```
Comment (dlech) [inline docs/tutorials/using-bluetooth-tethering.md]: What is the "charms menu"?

File: docs/tutorials/using-bluetooth-tethering.md [hunk 2]
```diff
@@ -135,11 +135,26 @@ If you are using an older version, please upgrade.
...
+        
+        4.  Your PC will search for Bluetooth devices. Connect to the device called *ev3dev*. Hit *Pair* on both your PC and the EV3                after verifying that the passcode is the same on both machines. It may take a while to connect.
```
Comment (dlech) [inline docs/tutorials/using-bluetooth-tethering.md]: You should only have to initiate the *Pair* on one device or the other, not both.

**Comment by dlech** [comment]
@demar42, this looks like a good start. If you want to copy my screenshots, go ahead. They should be saved into `images/windows/8` in this repository instead of linking to the existing images.

It would also be helpful if you are able to post a preview of your changes following the [instructions in the readme](https://github.com/ev3dev/ev3dev.github.io#previewing-your-changes-online).

File: docs/tutorials/using-bluetooth-tethering.md (hunk 1 above)
Comment (WasabiFan) [inline docs/tutorials/using-bluetooth-tethering.md]: It's the menu that appears in Windows 8 when you swipe in from the right or put your mouse in the upper-right corner.

**Comment by demar42** [comment]
I'm a bit of a github newbie and I'm exactly sure how to do the instructions in the readme. Do you mind simplifying them for me?

Thank you!

**Comment by dlech** [comment]
I don't know that I can make it any simpler. Don't worry if you can't figure it out.

**Comment by demar42** [comment]
Where would I execute the ./publish command? I don't have a ssh to my GitHub account

**Comment by dlech** [comment]
try `./publish.rb "https://github.com/demar42/ev3dev.github.io.git"`

```
# publish.rb
#
# This script publishes the ev3dev website to the gh-pages branch of your
# personal fork. This allows you to share a preview of your changes with others.
#
# Usage: ./publish.rb { <gh-user> | <gh-url> } [ --test '<command>' ] [ --no-fix-links ] 
#
# <gh-user> is your github user name. This is short for
# "git@github.com:<gh-user>/ev3dev.github.io.git"
#
# <gh-url> is the url for your fork on GitHub. For example, if you don't want
# to use ssh, then pass "https://github.com/<gh-user>/ev3dev.github.io.git"
# as the argument.
#
# --test '<command>' will run <command> in a shell and return the result. The
# working directory will be a temporary directory containing the fixed up files.
#
# --no-fix-links will prevent modification of links in anchor and img elements.
#
# Use BASENAME environment variable to override basename. Use "@FULL_PATH@" as a
# placeholder for the full path to the temporary directory that is created.
```

**Comment by WasabiFan** [comment]
What's the status of this? I just tested it locally and the formatting seems fine to me.

**Pull request closed without merging**